sirepo/template/elegant.py

```diff
@@ -1453,3 +1456,23 @@ def _sdds_init():
     _x = getattr(_s, "SDDS_LONGDOUBLE", None)
     _SDDS_DOUBLE_TYPES = [_s.SDDS_DOUBLE, _s.SDDS_FLOAT] + ([_x] if _x else [])
     _SDDS_STRING_TYPE = _s.SDDS_STRING
+
+
+def stateless_compute_log_to_html(data):
+    r = simulation_db.simulation_run_dir(
+        PKDict(
+            simulationType=data.simulationType,
+            simulationId=data.simulationId,
+            report="animation",
+        )
+    )
+    text = pkio.read_text(r.join(data.filename))
+    t = pygments.highlight(
```

Comment:
text and t temporaries unnecessary. They don't shorten the code, and it's not too deep nesting.